src/main/java/com/minecolonies/items/ItemAchievementProxy.java

```diff
@@ -0,0 +1,27 @@
+package com.minecolonies.items;
+
+
+/**
+ * This is a proxy item class for achievements.
+ *
+ * Achievements can't have custom icons except using an item or block as proxy.
+ */
+public class ItemAchievementProxy extends AbstractItemMinecolonies
```

Comment:
This point is done.

I moved the creativetab assignment into each item itself.

If you have another solution, please tell me

@cltnschlosser